magento/magento2
# [Backport] Fixes reverted for remove space when only one country in drop-down on both cart …

### Description

In Magento 2.1.X version having issue of extra space on both cart and checkout page for country dropdown. Even only one country is selected from admin.
I have applied fixes in core to remove this space issue from all version of 2.1.X


### Fixed Issues (if relevant)

1. #2146: Countries drop-down is empty



### Manual testing scenarios

1. Go in admin >> store >> configuration >> general under country options select one country in "Allow country" multi select drop-down and save it and then add product and move on both cart and checkout page there you will see country drop-down which contain no blank field or extra space before country which is select-able from admin.


### Contribution checklist
